common/prettify.pm

```diff
@@ -1497,6 +1498,7 @@ sub Setup_Handler ($)
       if ("$totals->{GIT_CHECKEDOUT_ACE}" eq "Matched")
       {
         $totals->{GIT_CHECKEDOUT_ACE} = $sha;
+        push(@commits, "ACE:" . $sha);
```

Comment:
Couldn't this be done more generic, in the log parsing, for each git clone create a `GIT_CHECKEDOUT_` with the repo name as upper case and in this section look through the array and push the string without `GIT_CHECKEDOUT_` prefix. That would also give us the MPC git hash and when additional repositories are used they are all added

Reply:
I'd like to avoid expanding the scope of this PR - if autobuild is already using a design based on `GIT_CHECKEDOUT_ACE` then let's keep that for now.